DEPLOYMENT_GUIDE.md

```
@@ -459,12 +473,14 @@ redis-cli slowlog get 10
 ## 📞 SUPPORT & CONTACTS
 
 ### Emergency Contacts
+
 - **Platform Team**: platform-team@neonpro.com.br
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)